CenterForOpenScience/osf.io
# Mass spamify [ENG-2827]

## Purpose

To get rid of a ton of spam at once

## Changes

Adding management command, tests, and admin app functionality
## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify
- Verify

What are the areas of risk?N/A

Any concerns/considerations/questions that development raised?N/A

## Documentation

N/A

## Side Effects

N/A

## Ticket

https://openscience.atlassian.net/browse/ENG-2827
